boltdesignsystem/bolt
# DS-372: Standardize Twig and Schema (Link)

## Jira

[https://pegadigitalit.atlassian.net/browse/DS-372](https://pegadigitalit.atlassian.net/browse/DS-372)

## Summary

Standardize Twig and Schema for Link

## Details

Standardize Twig and Schema for Link. `shadow-root` is removed intentionally, as it is no longer needed. The snapshots have been updated because the default values of properties no longer need to be printed. For more detailed information, please see the initial ticket, as well as the associate TWIG and Schema tickets. 

## How to test

Check out the tests and demos and make sure nothing is a breaking change in the code.

## Release notes

Deprecate `isHeadline` prop in favor or `is_headline`.
